### PR TITLE
feat: setLoginTimeout implementation

### PR DESCRIPTION
**Pull request by karenc-bq**
### Summary

Implement setLoginTimeout as suggested in #509 

### Additional Reviewers

@jrobe

### By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java [hunk 1]
```diff
@@ -284,12 +295,19 @@ public void setLogWriter(final PrintWriter out) throws SQLException {
 
   @Override
   public void setLoginTimeout(final int seconds) throws SQLException {
-    throw new SQLFeatureNotSupportedException();
+    if (seconds <= 0) {
```
Comment (sergiyvamz) [inline wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java:298]: I'd validate user input here.

File: wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java [hunk 2]
```diff
@@ -284,12 +295,19 @@ public void setLogWriter(final PrintWriter out) throws SQLException {
...
+      return;
+    }
+    loginTimeout = seconds;
   }
 
   @Override
   public int getLoginTimeout() throws SQLException {
-    throw new SQLFeatureNotSupportedException();
+    if (loginTimeout <= 0) {
```
Comment (sergiyvamz) [inline wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java:306]: Probably we shouldn't raise an exception in the getter method.

File: wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java [hunk 3]
```diff
@@ -153,6 +154,16 @@ public Connection getConnection(final String username, final String password) th
     if (!StringUtils.isNullOrEmpty(this.targetDataSourceClassName)) {
 
       final DataSource targetDataSource = createTargetDataSource();
+      try {
```
Comment (sergiyvamz) [inline wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java:157]: I'm not sure about this try-catch block. What's the purpose? Does setLogintimeout() require a special handling?

Comment (karenc-bq) [reply wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java]: this calls the setLoginTimeout method in the target datasource, which may potentially throw a SQLException due to database access errors

**Pull request merged**